### ISSUE TITLE
Modelling wind-terrain interactions in FDS, with complex terrain and intermittent wind ...

### ISSUE THREAD

**Issue by brunoguillaume** [firemodels/fds#10274]
Let us forget the aspect on intermittent wind for now, and concentrate only on the aspect of coherence of stationary wind on complex terrain.
Here attached is the downslope case, with possibility to force either with "wall of wind" or with "Monin-Obukhov Similarity" (I run in 6.7.5).
Looking at the wind profiles in the DEVC, you will see the weird behavior of wind velocity in the lowest atmospheric levels above terrain, after going >30m downslope.

[slopem10_new.zip](https://github.com/firemodels/fds/files/7936273/slopem10_new.zip)

**Comment by mcgratta** [comment]
I coarsened the grid to 0.5 m, removed the particles, and applied a steady wind field (not a wall of wind). This is what I see. Let's start with something like this. First, I am not interested in the wall of wind approach, so let's just remove that from consideration. Next, I'm not sure that this configuration is the best for studying fundamental behavior. I think it would be better to look at a ridge or hill where the boundaries are set back from the region of interest. 
https://user-images.githubusercontent.com/11333911/151066895-0eb65339-91af-426c-8001-3057ab989cdf.mp4
All this being said, what do you mean by "weird behavior"?

**Comment by brunoguillaume** [comment]
Thanks for the important simplifications on the test case: I use them too in this new file:
[slopem10_2.zip](https://github.com/firemodels/fds/files/7940178/slopem10_2.zip)

I did 4 simulations on this case: 
- sim1: cst Z-profile (value of 20m/s for all z) as input to &WIND (same as your run, if I understand correctly) : 
https://user-images.githubusercontent.com/51914955/151131957-4fa9d870-9728-43bc-bd00-8adef63f2c7e.mp4
- sim2: cst Z-profile as input "wall of wind" (after initial transition phase of 15s) : 
https://user-images.githubusercontent.com/51914955/151131991-832075c9-1f23-47b2-b2d8-c6cc9fc347c6.mp4
- sim3: log Z-profile as input to &WIND (more exactly: MO-neutral): 
https://user-images.githubusercontent.com/51914955/151132037-fd1890f8-6d69-4f0c-9054-16a177e7201f.mp4
- sim4: log Z-profile as input "wall of wind", using a RAMP_V_Z with value 0 at altitude of top of slope (after initial transition phase of 15s) : 
https://user-images.githubusercontent.com/51914955/151132060-973b5d5e-4fe4-4bae-88d9-ab27bf456c66.mp4

I put on purpose a pure downslope situation, and not a uphill/top hill/downhill situation, since I am sure that the wind from FDS reproduces well the eddies just in the wake after passing the top of hill, no question about that.

The importance of having a coherent log profile in the first cms/meters of atmosphere for surface fires in wildfires, even in a downslope situation, is to reproduce dynamically the equivalent of the Rothermel "Wind Recution Factor": a fuel located on the ground (grass, litter...) would not receive the same wind intensity as a shrub, or as low branches of canopies. If we use the 2m-height wind value for those latter, we would need to reduce this value for the surface fuel.

I rather want to see in a pure downhill situation if a log profile is conserved down the slope, which I guess would be the correct physical surface layer, because of the same atmospheric processes acting on the air mass at each point when going downhill:
1/ soil rugosity inducing a wind=0 on the ground and a shear in the lowest first z-meters above the ground;
2/ gravity compensated by flottability
( of course with unavoidable small eddies occuring from the steppy topography)
In stable atmosphere there might probably be intensification of the gliding of air mass down the slope (don't know if we would then expect intensification of wind when going downslope...) but here we are in neutral case.

I run initially the "wall of wind" only, since I got clear that "&WIND" will not be maintained in versions>=6.7.6.

The "weird behavior" is on "wall of wind" simulations that, even with a wind having a constant z-profile (sim2), we see that, starting from top of slope, the more we go downslope, the more we loose wind speed intensity in the very first meters above terrain. This is worse for the more physical log z-profile (sim4).
It is less of a problem in "&WIND" simulations, maybe still a bit of a problem on the log z-profile (sim3). Is it that the nudging is done based on a local-z assumption in respect to local ground_level, rather than on a global-z assumption ?

Since &WIND is not maintained, what would be the solution for "wall of wind"? Any possibillity to add a contribution of gravity/flottability (and also surface rugosity) not as boundary conditions but as fields in the free flow equations ?

**Comment by mcgratta** [comment]
Where did you get the impression that &WIND is not going to be maintained? Actually, WIND is the newer, preferred method. It is the "wall of wind" that we do not want to develop any further, although one will always be able to create a "wall of wind" in any FDS simulation. 

As for these various cases -- I do not think that this simple downslope is a good way to test ideas. In fact, I'd rather compare with actual wind measurements or a simple case in which the boundaries are far from the hill. It is unnatural to impose a wind condition at the top of a hill. In reality, the wind field over the hill develops far upwind.

**Comment by rmcdermo** [comment]
I do not like where this is heading.  I am all for doing validation of the boundary layer profile.  But this is not how we do it. 
 Please send us a reference to a paper with the mathematical statement or experimental results for the problem we are trying to solve.

**Comment by brunoguillaume** [comment]
Very sorry, I got confused by the following sentence in the release notes of v6.7.6 : "Mean forcing for wind has been removed, but the basic input parameters for wind remain the same.". I thought the "mean forcing" did stand for &WIND.
Then this is good news for me that the &WIND can be used !
@Randy : due to my misunderstanding, I was dispapointed to have to use "wall of wind" in my context, also because of the slow convergence time to reach stationary state in my domain. &WIND will be ok, no need to revisit the surface layer parameterization.
